README.md

```diff
@@ -0,0 +1,146 @@
+# geocoder-argis-geojson
+
+Isomorphic geocoding for reuse across our JavaScript libraries. This library is built to match the API of [Isomorphic Mapzen Search](https://github.com/conveyal/isomorphic-mapzen-search/) in order to facilitate a quick conversion of our applications to use ESRI's geocoder.  Read the ESRI docs [here](https://developers.arcgis.com/labs/rest/get-an-access-token/). Coordinates must be anything that can be parsed by [lonlng](https://github.com/conveyal/lonlng).
+
```

Comment:
Is it a joke that this redirects to https://github.com/conveyal/lonlat?

Reply:
We used to call it that, but then it got changed to what it is now.  @trevorgerhardt can give some more history if need be.  :)

Reply:
I love that a repo with a snarky readme about lat/lon normalization has a redirect!